Umweltzone/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,327 @@
+        Portanto, gostaria de lhe pedir pessoalmente que libere as geo-coordenadas sob uma <b>licença gratuita</b>.
+        Para facilitar o processamento de dados, os seguintes <b>formatos de arquivo</b> são adequados:
+        ESRI Shapefiles, CSV, GeoJSON.<br/>
+        <br/>
+        Atenciosamente</p>
+        <p>----<br />
+        <br/>
+        Sehr geehrte Damen und Herren,<br/>
+        <br/>
+        ich nutze die kostenfreie und quelloffene Anwendung &quot;Umweltzone&quot; [1],
+        mit der ich mich über den Verlauf von Umweltzonen und individuelle
+        Regelungen in verschiedenen Städten auf meinem Android-Gerät
+        informieren kann. Ich betrachte diese Anwendung als nützliche
+        Erweiterung zum bestehenden Informationsangebot vieler Städte.<br />
+        <br/>
+        Leider kann die <b>Umweltzone %1$s</b> nicht dargestellt werden. Es fehlen
+        die Geokoordinaten des Umweltzonenumrings. Daher möchte ich Sie hiermit
+        persönlich bitten, die Koordinaten unter einer <b>freien Lizenz</b> zu
+        veröffentlichen. Um eine Weiterverarbeitung zu ermöglichen, eignen
+        sich folgenden <b>maschinenlesbare Dateiformate</b>:
+        ESRI Shapefiles, CSV, GeoJSON.<br/>
+        <br/>
+        Mit freundlichen Grüßen</p>
+
+        <p>___<br/>
+        [1]: <a href="https://play.google.com/store/apps/details?id=de.avpptr.umweltzone">https://play.google.com/store/apps/details?id=de.avpptr.umweltzone</a></p>]]>
+    </string>
+
+    <!-- TraceDroid -->
+    <string name="trace_droid_dialog_title">
+        Enviar relatório de falhas do Umweltzone &#8230;
+    </string>
+    <string name="trace_droid_dialog_message">
+        Foi detectado um problema com o Umweltzone.
+        Ficaríamos muito gratos se você pudesse enviar um relatório de falhas
+        para que a origem do problema pudesse ser identificada.
+    </string>
+    <string name="trace_droid_button_title_no">Não</string>
+
+    <!-- SnackEngage -->
+    <string name="snack_engage_rate_title">Tudo bem?</string>
+    <string name="snack_engage_rate_action">Por favor, avalie o app</string>
+    <string name="snack_engage_beta_testing_title">Usuário avançado?</string>
+    <string name="snack_engage_beta_testing_action">Torne-se um testador beta</string>
+
+    <!-- Settings -->
+    <string name="settings_activity_title">Configurações</string>
+    <string name="settings_category_title_data_privacy">Privacidade dos dados</string>
+    <string name="settings_title_google_analytics">Envie informações anônimas para análises estatísticas</string>
+    <string name="settings_summary_google_analytics">
+        Permitir a transmissão de estatísticas de uso e relatórios de falhas para o Google Analytics. Esses
+         dados são anônimos e usados pelo desenvolvedor do Umweltzone para investigar os
+         erros e assim, melhorar continuamente o aplicativo.
+    </string>
```

Comment:
Please harmonize the indentation here, e.g.:

``` xml
<string name="settings_summary_google_analytics">
    Permitir a transmissão de estatísticas de uso e relatórios de falhas para o Google
    Analytics. Esses dados são anônimos e usados pelo desenvolvedor do Umweltzone para
    investigar os erros e assim, melhorar continuamente o aplicativo.
</string>
```

Reply:
Sure! Done!